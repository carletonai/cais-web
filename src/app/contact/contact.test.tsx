import React from 'react';
import { render } from '@testing-library/react';
import ContactPage from './contact';

test('renders ContactPage correctly', () => {
  const { asFragment } = render(<ContactPage />);
  expect(asFragment()).toMatchSnapshot();
});

