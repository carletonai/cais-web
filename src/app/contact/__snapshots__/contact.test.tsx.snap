// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ContactPage correctly 1`] = `
<DocumentFragment>
  <div
    class="min-h-screen p-8"
  >
    <h1
      class="text-4xl font-bold mb-6"
    >
      Contact Us
    </h1>
    <p
      class="text-lg"
    >
      Get in touch with the CAIS team.
    </p>
  </div>
</DocumentFragment>
`;
